dask/tests/test_distributed.py

```diff
@@ -75,3 +75,20 @@ def test_local_get_with_distributed_active(c, s, a, b):
     y = delayed(inc)(2).persist(get=dask.async.get_sync)
     yield gen.sleep(0.01)
     assert not s.task_state # scheduler hasn't done anything
+
+
+
+def test_to_hdf_distributed(loop):
+    from ..dataframe.io.tests.test_hdf import test_to_hdf
+    with cluster() as (s, [a, b]):
+        with distributed.Client(s['address'], loop=loop) as c:  # noqa: F841
```

Comment:
You could also remove the `noqa` and just use delete the `as c`. Same for below.